src/models/user.ts

```diff
@@ -319,4 +319,23 @@ export const Users: UserCollection = {
 
     return null;
   },
+
+  async addActiveDayEvent (client, user, origin) {
+    const [exists] = await client('user_account_active_day')
+      .where({
+        user_account_id: user.id,
+         origin,
```

Comment:
how come this inconsistent indentation isn't caught by the linter?

Reply:
cos palantir...